a.sh

```
@@ -0,0 +1,87 @@
+    --src-l2-chain-id $(cast chain-id -r $OP_RPC_URL) \
+    --dst-l2-chain-id $(cast chain-id -r $BASE_RPC_URL) \
+    --src-l2-http-path $OP_RPC_URL \
+    --dst-l2-http-path $BASE_RPC_URL \
+    --src-contract-address 0x1BDD24840e119DC2602dCC587Dd182812427A5Cc \
+    --src-storage-slot 0x0000000000000000000000000000000000000000000000000000000000000002 \
+    --l1-http-path $ETH_SEPOLIA_RPC_URL \
+    --l1-registry-address $SETTLEMENT_REGISTRY &> $OUTFILE
+
+echo "Proof output written to $OUTFILE"
+
+export PROOF=$(tail -n 1 $OUTFILE)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)